release/version.py

```diff
@@ -47,7 +48,7 @@ def commit(length: int = 7) -> str:
     return p.stdout.decode("utf-8").strip()
 
 
-def semver(git_version: str | None = None) -> str:
+def semver(git_version: Optional[str] = None) -> str:  # noqa: UP007 - RHEL8 has python 3.8
```

Comment:
we should create a card to make release run in container so no dependencies on ci host

Reply:
Done https://issues.redhat.com/browse/APPSRE-10403